near-rs/integration-tests/src/tests.rs

```diff
@@ -3,18 +3,39 @@ use std::{
     time::{Duration, SystemTime, UNIX_EPOCH},
 };
 
-use near_units::parse_near;
 use serde::Deserialize;
 use serde_json::json;
-use workspaces::{Account, Contract};
+use workspaces::{Account, Contract, Worker, network::Sandbox};
 
 #[derive(Deserialize, Debug, PartialEq)]
 pub struct Grant {
+    owner: String,
     grantee: String,
     data_id: String,
     locked_until: u128,
 }
 
+async fn create_account(new_account_id: &str, worker: &Worker<Sandbox>) -> anyhow::Result<Account> {
+    Ok(worker.dev_create_account().await?
+        .create_subaccount(new_account_id)
+        .transact()
+        .await?
+        .into_result()?)
+}
```

Comment:
This doesn't seem to be working as I expect. I get all the accounts with the same signer.

Reply:
Nevermind, I realized I don't need full account, only keys. Generated random ones.